compose/gettingstarted.md

```diff
@@ -26,6 +26,7 @@ Define the application dependencies.
 
 2.  Create a file called `app.py` in your project directory and paste this in:
 
+        #!/usr/bin/env python
```

Comment:
Was there a reason for adding this? This file will not be used as an executable script, but is called in the dockerfile with;

```dockerfile
CMD ["python", "app.py"]
```

(so it's run by python, and not in a shell)

Reply:
Yes, at one point Werkzeug/Flask had an issue with its reloader on Windows because volumes mount all files as 777 since Windows doesn't have an executable bit. This is no longer needed because it was fixed in Werkzeug. (That `CMD` line may have been changed to fix it too, can't remember.)

Reply:
Oh! For some reason, github opened on the changes tab, and I didn't see your earlier comment about the change that was already merged

Alright; let's close this one; thanks!